docs/configuration-discovery-instance-filters.md

```diff
@@ -0,0 +1,101 @@
+# Configuration: instance filters
+
+Sometimes it is desirable to exclude particular hosts from discovery.
+It may be the case for large, complex environments where auxiliary nodes like gh-ost, Debezium, Tungsten, etc., are present.
+
+A node can be excluded by specifying its host's name regex pattern. When the Orchestrator detects such a node and its host name matches one of the given patterns, the node is skipped during the discovery process:
+
```

Comment:
To skip a node during the Discovery process, specify the host name using a regex pattern. If the host name matches the pattern, Orchestrator skips the node and continues the process.